### ISSUE TITLE
Trouble with installation on Ubuntu 

### ISSUE THREAD

**Issue by catweis** [DeepMicroscopy/Exact#101]
First of all, thank you very much for this excellent tool.
Unfortunately, I have trouble installing it on a Linux machine (Ubuntu 22); not due to the Ubuntu version.
I have cloned your repo, modified the env files in the subfolder exact/exact, and then started the installation via:
 **sudo docker-compose -f docker-compose.yml up -d --build**
By doing so, there is an error with a requirements.txt file. I have copied that one from the subfolder exact to the main folder.
However, there is still this error:
  **The command '/bin/sh -c pip3 install -r requirements.txt' returned a non-zero code: 137
  ERROR: Service 'web' failed to build : Build failed**

**Comment by ChristianMarzahl** [comment]
Dear @catweis,

First of all, you have a very cool profile picture.
Second: Could you provide me with more details regarding your error message? Could you let me know why you copied the requements.txt?

With kind regards

**Comment by catweis** [comment]
Dear @ChristianMarzahl,
Thank you. As a pathologist, it was somehow relatively nearby to choose a histology image.
Regarding your questions: I have copied the settings file due to an error message mentioning that it is missing. Furthermore, it is mentioned in your installation guide:
"Copy and rename settings.py.example to settings.py in the exact folder."
Therefore, I have copied it. However, there are two exact folders in the repo? Repo/exact and Repo/exact/exact.
Interestingly, installing the production version does work. 
For testing etc. the development version would be much easier. In addition, we maybe want to add features.
